openfl-tutorials/interactive_api/Tensorflow_MNIST/workspace/Mnist_Classification_FL.ipynb

```diff
@@ -0,0 +1,699 @@
+{
```

Comment:
Validate this file: `Unreadable Notebook: Mnist_Classification_FL.ipynb NotJSONError('Notebook does not appear to be JSON: \'{\\n "cells": [\\n {\\n "cell_type": "m...')`